config-release/values-schema.yaml

```diff
@@ -0,0 +1,17 @@
+#@data/values-schema
+---
+#@overlay/match missing_ok=True
+#@overlay/match-child-defaults
+#@schema/desc "Configuration explicitly for developing secretgen-controller"
+dev:
+  #@schema/desc "Location of kapp-controller image"
+  image_repo: "docker.io/ysethiya/carvel-kapp-controller"
+  #@schema/desc "Development version"
+  kapp_controller_version: develop
+  #@schema/desc "Comma separated list of supported architectures"
+  platform: "linux/amd64,linux/arm64"
+  #@schema/desc "image push flag"
```

Comment:
Let's copy the descriptions from SGC: 

https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/config/dev/values-schema.yml#L7